clippy_lints/src/if_not_else.rs

```diff
@@ -48,13 +48,6 @@ declare_clippy_lint! {
 
 declare_lint_pass!(IfNotElse => [IF_NOT_ELSE]);
 
-fn is_zero_const(expr: &Expr<'_>, cx: &LateContext<'_>) -> bool {
```

Comment:
I think that we can remove this, as we already have the same function in utils.

Reply:
> I think that we can remove this, as we already have the same function in utils.

I'm not sure I follow: your comment seems to precisely point to code that does this removal.

Reply:
Oh, sorry. My PR reviewing UI in neovim was highlighting these two diffs as green, so I thought you added it. Sorry for the confusion.

Reply:
No problem, I wondered for a few seconds if I had done something weird while juggling with my patch stacks in jj.